docs/SealCLI.md

```diff
@@ -51,7 +51,7 @@ Symmetric key: e39651e5aa01949ba5174c67a2c37f58ee8217392ba2275a5789f0ac2c3540d8
     The encryption is randomized, so the output will be different each time you run the command, even with the same input message and keys.
 
 !!! info
-    The output includes the BCS-encoded encrypted object and the symmetric key used to encrypt the message. You can share the encrypted object (for example, on-chain or via Walrus), but keep the symmetric key secret - anyone with access to it can decrypt the message.
+    The output includes the Hex encoded BCS serialized encrypted object and the symmetric key used to encrypt the message. You can share the encrypted object (for example, on-chain or via Walrus), but keep the symmetric key secret - anyone with access to it can decrypt the message.
```

Comment:
```suggestion
    The output includes the Hex-encoded, BCS serialized encrypted object and the symmetric key used to encrypt the message. You can share the encrypted object (for example, on-chain or via Walrus), but keep the symmetric key secret - anyone with access to it can decrypt the message.
```